### ISSUE TITLE
quickstart: assumes implicit knowledge of Makefile conventions

### ISSUE THREAD

**Issue by lucab** [monitoring-mixins/docs#9]
First of all, thanks for the project and the docs!

Starting from no knowledge about monitoring mixins, I was trying to write some quick dashboarding mixin for a project of mine, by following the docs in this repo (as mirrored on https://monitoring.mixins.dev/).

I somehow got stuck on the step where it runs `make`. This assumes that the mixin-author already has a lot of knowledge about what they are supposed to do and what are the conventions around such Makefile.

I unfortunately had neither of those, so I was really tempted to just blindly copy the magic incantation from the first Makefile I could find (e.g. the node-exporter [one](https://github.com/prometheus/node_exporter/blob/35bfe455df207930537e2df4b93536290f239273/docs/node-mixin/Makefile)).

I wasn't really convinced by that, and I ended up just running a minimal `mixtool generate` which seems to take care of the same logic.

For newcomers like me, it would be great to write down what is the interface for properly performing artifacts generations (and how to lay the mixins on disk so that generators are happy).

**Comment by brancz** [comment]
I think this is very good idea! Since you just went through it, do you think you could write up what you think you would have expected to see and contribute that? :)

**Comment by lucab** [comment]
For reference, I was starting from zero and trying to assemble this https://github.com/coreos/zincati/tree/v0.0.12/contrib/monitoring-mixins.

I ended up basically just using `mixtool generate dashboards -d generated/dashboards/ mixin.libsonnet` but it's unclear to me whether that tool is actually still maintained.

**Comment by brancz** [comment]
Yeah the tool hasn't really evolved in a while.

@metalmatze do you think we can maybe pick mixtool back up one of these days and discuss what it might look like going forward?

**Comment by metalmatze** [comment]
Was literally discussing this in the last week with @sh0rez.
Didn't see this thread until now. 🤔

Tom wants something like this for Tanka. I think there should be a standalone tool. So I'm happy to move forward with it.

**Comment by tombrk** [comment]
Indeed, working on a standalone tool! Will open source as soon as it kinda works :)

**Comment by metalmatze** [comment]
Couldn't we make this a continuation of this project rather than recreating with a new one?

**Comment by brancz** [comment]
I feel similar, why create something new? The mixtool exists ...? Anyways, would love to collaborate! :)
